webpack.config.js

```diff
@@ -0,0 +1,9 @@
+function buildConfig() {
+    const env = process.env.NODE_ENV;
+    if (env === 'dev' || env === 'prod') {
```

Comment:
Also this kind of strings is better to use as constant like this.
```js
const DEV = 'dev';
```

Reply:
This is done